EntityFramework/HomeWork3/Contracts/ICategoryRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FoodOrderingSystem.Models;
+using System.Collections.Generic;
+
+namespace FoodOrderingSystem.Сontracts
+{
+    public interface ICategoryRepository
+    {
+        Category GetCategoryById(int id);
+        List<Category> GetAllCategories();
+        void InsertCategory(Category category);
```

Comment:
maybe remove word "Category" from methods name cause the are in "Category" repo  ?